.cursor/rules/strings.mdc

```diff
@@ -0,0 +1,4 @@
+---
+description: String literals should always be defined in and referenced from @enus.json
```

Comment:
Curious what `@enus` is?

Reply:
@swansontec this is the way of referencing a file for context, so the agent doesn't need to go spend tool calls to find the file